hawkbit-repository/hawkbit-repository-core/src/main/resources/hawkbit-repository-defaults.properties

```diff
@@ -94,4 +94,8 @@ hawkbit.server.tenant.configuration.implicit-lock-enabled.keyName=implicit.lock.
 hawkbit.server.tenant.configuration.implicit-lock-enabled.defaultValue=true
 hawkbit.server.tenant.configuration.implicit-lock-enabled.dataType=java.lang.Boolean
 
+hawkbit.server.tenant.configuration.actions-cleanup-on-quota-hit-percent.keyName=actions.cleanup.onQuotaHit.percent
```

Comment:
maybe better move it up together with the other action-cleanup property, in order to be at one place - no need to be in order of introduction